src/ja/server/scheduler/default_policies.py

```diff
@@ -1,26 +1,42 @@
 from abc import ABC, abstractmethod
-from ja.common.job import Job
+from copy import deepcopy
+from ja.common.job import Job, JobPriority
+from ja.common.work_machine import ResourceAllocation
 from ja.server.database.database import ServerDatabase
+from ja.server.database.types.job_entry import DatabaseJobEntry
 from ja.server.database.types.work_machine import WorkMachine
 from ja.server.scheduler.algorithm import CostFunction, JobDistributionPolicy
-from typing import List, Optional, Tuple
+from typing import Dict, List, Optional, Tuple
+
+import datetime as dt
 
 
 class DefaultCostFunction(CostFunction):
     """
     The default cost function provided by JobAdder.
     """
+    _base_costs: Dict[JobPriority, Tuple[float, float]] = {
+        JobPriority.URGENT: (0, -1),
+        JobPriority.HIGH: (500, -1),
+        JobPriority.MEDIUM: (1000, -0.5),
+        JobPriority.LOW: (2000, -0.1)
```

Comment:
If you look more carefully, you'll see the order never changes because multipliers are also different.

That being said, there is no real reason for different modifiers. I left that for further tweaking if needed.

Reply:
The order of individual jobs can change.
Imagine a scenario with an old low-priority job and a new high-priority job, with the current effective priority of the older job being higher.
In the beginning the older job will be scheduled first but at some point the newer job will have a higher priority and therefore overtake it.
I think this can lead to problems in combination with blocking jobs.

Reply:
I don't see problems with blocking but I'll change them to be equal.